dvc/config.py

```diff
@@ -126,6 +126,7 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     SECTION_CORE_INTERACTIVE = "interactive"
     SECTION_CORE_ANALYTICS = "analytics"
     SECTION_CORE_CHECKSUM_JOBS = "checksum_jobs"
+    SECTION_CORE_NFS_LOCK = "nfs_lock"
```

Comment:
The name is misleading, as only old nfs versions are affected. Might be able to come up with something better.

Reply:
`hardlink_lock`?

Reply:
@Suor Have thought about that, but I'm worried that it will confuse people 🙁 Though, maybe better than nfs.

Reply:
This is very specific thing people won't use without consulting docs or us. `nfs_lock` might also be confusing when used for something else than NFS or when it is not used for NFS :)